idl/github.com/uber/cadence/shared.thrift

```diff
@@ -254,6 +258,20 @@ struct TaskList {
   20: optional TaskListKind kind
 }
 
+enum EncodingType {
```

Comment:
Considering we are not exposing GetWorkflowExecutionRawHistory api to clients in this PR, I would recommend to not expose this on public thrift interfaces.  Especially how we expose headers like encoding back to clients needs some discussion.